jvm/src/test/scala/uk/gov/homeoffice/drt/db/dao/QueueSlotDaoTest.scala

```diff
@@ -0,0 +1,133 @@
+package uk.gov.homeoffice.drt.db.dao
+
+import org.scalatest.BeforeAndAfter
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import specs2.arguments.sequential
+import uk.gov.homeoffice.drt.db.TestDatabase
+import uk.gov.homeoffice.drt.model.CrunchMinute
+import uk.gov.homeoffice.drt.ports.PortCode
+import uk.gov.homeoffice.drt.ports.Queues.EeaDesk
+import uk.gov.homeoffice.drt.ports.Terminals.T2
+import uk.gov.homeoffice.drt.time.{SDate, UtcDate}
+
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.DurationInt
+
+class QueueSlotDaoTest extends AnyWordSpec  with Matchers with BeforeAndAfter {
+  sequential
+
+  private val db = TestDatabase.db
+
+  import TestDatabase.profile.api._
+
+  val dao: QueueSlotDao = QueueSlotDao()
+
+  println(dao.table.schema.createStatements.mkString(";\n") + ";")
```

Comment:
println here

Reply:
Yeah I'll move these to a utility function :+1: